### ISSUE TITLE
[DEP #1] build-in print statement

### ISSUE THREAD

**Issue by PeterlitsZo** [defer-lang/defer#1]
I thinnk if the print is a build-in statement will make it better.

like this below:
``` defer
print "hello"
```

and the output will be like this:
``` defer
>>> print "hello"
  | hello
>>> print "hello\nworld!!!"
  | hello
  | world!!!
```

**Comment by Trigger-FK** [comment]
Many language print sentences with parenthesis , but this doent use it, right?

**Comment by PeterlitsZo** [comment]
> Many language print sentences with parenthesis , but this doent use it, right?

Yes, it does. In Python 2.0 they were still using the way like `print "bala"`. But after Python 2.0 (in Python 3.0 age), they use `print("bala")` instead of the way without parenthesis. Because Python function will be called by `function-name(function-argument...)`, and the parenthesis make it looks better and easy to use.

But this print I want to make it using like Lisp - function will called by outside parethesis, like `(function-name function-argument...)`.

I do think it is a good idea and easy to use. For me, I think it will be a good idea.

**Comment by Trigger-FK** [comment]
明白了！ I feel that this idea is good.
